package.json

```diff
@@ -43,6 +43,7 @@
     "bedrock-passport": "4.0.2 - 5.x",
     "bedrock-permission": "^2.5.0",
     "bedrock-mongodb": "3.x - 6.x",
+    "bedrock-security-context": "^3.0.0",
```

Comment:
should be a peerDep for bedrock-credentials-context@1 and it should be explicitly required somewhere in the test suite as with bedrock-security-context.

Reply:
Fixed here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/20/commits/61ad2df6fb5217e33da13e5552f80d2958dccf0f

```js
  "peerDependencies": {
    "bedrock": "1.12.1 - 3.x",
    "bedrock-account": "^2.1.0",
    "bedrock-express": "2.0.8 - 3.x",
    "bedrock-https-agent": "^1.0.1",
    "bedrock-passport": "4.0.2 - 5.x",
    "bedrock-permission": "^2.5.0",
    "bedrock-mongodb": "3.x - 6.x",
    "bedrock-credentials-context": "^1.0.0",
    "bedrock-server": "^2.3.2",
    "bedrock-validation": "^4.1.0",
    "bedrock-zcap-storage": "^1.0.0"
  }, 
```

and in `/test/package.json`
```js
  "dependencies": {
    "apisauce": "^1.1.1",
    "bedrock": "^3.1.0",
    "bedrock-account": "^2.1.3",
    "bedrock-express": "^3.1.0",
    "bedrock-https-agent": "^1.0.1",
    "bedrock-identity": "^7.0.1",
    "bedrock-jsonld-document-loader": "^1.0.1",
    "bedrock-mongodb": "^6.0.2",
    "bedrock-passport": "^5.0.2",
    "bedrock-permission": "^2.5.4",
    "bedrock-credentials-context": "^1.0.0",
    "bedrock-server": "^2.4.1",
    "bedrock-test": "^5.0.0",
    "bedrock-validation": "^4.2.0",
    "bedrock-vc-issuer": "file:..",
    "bedrock-zcap-storage": "^1.2.0",
    "sinon": "^8.0.4"
  }
```